src/build.js

```diff
@@ -69,6 +69,12 @@ const followBuild = async ({ appSlug, buildSlug, client }, options = {}) => {
       throw new Error(`Build ${appSlug}/${buildSlug} aborted`);
     }
 
+    // If the build is completed and there are no more log chunks
+    if (!!attributes.build_finished && response.data.log_chunks.length === 0) {
```

Comment:
It looks like the responses from bitrise changed. Even when the builld has completed, the `response.data.timestamp` returns non-null

Reply:
Is it worth changing the loop condition in that case?

Reply:
I was too scared to break something. It's probably possible to refactor but I thought better safe than sorry